Gemfile.lock

```diff
@@ -0,0 +1,83 @@
+PATH
+  remote: .
+  specs:
+    just-the-docs (0.4.0.rc3)
+      jekyll (>= 3.8.5)
+      jekyll-seo-tag (>= 2.0)
+      rake (>= 12.3.1)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    colorator (1.1.0)
+    concurrent-ruby (1.1.10)
+    em-websocket (0.5.3)
+      eventmachine (>= 0.12.9)
+      http_parser.rb (~> 0)
+    eventmachine (1.2.7)
+    ffi (1.15.5)
+    forwardable-extended (2.6.0)
+    google-protobuf (3.21.12)
+    http_parser.rb (0.8.0)
+    i18n (1.12.0)
+      concurrent-ruby (~> 1.0)
+    jekyll (4.3.1)
```

Comment:
Should `Gemfile.lock` correspond to the standard build on GH Pages? If so, I think it should specify Jekyll 3.9.2.

If I want to build the theme docs locally, I change the gemspec to insist on `jekyll ~> 3.9`. The resulting Gemfile.lock then has different versions of quite a lot of the other gems.

Users who want to build the theme docs with Jekyll 4 can then simply use `bundle update` instead of `bundle install`. 

If one is already using Jekyll 4, and wants to switch to Jekyll 3, I think one needs to delete `Gemfile.lock` (since `bundle update` doesn't seem to look for older versions of other gems).

Reply:
I agree to use Jekyll 4 to build the theme docs.